WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -25,7 +29,7 @@ public final class SingleOrderScreen: ScreenObject {
         tabBar = try TabNavComponent(app: app)
 
         try super.init(
-            expectedElementGetters: [ summaryCellGetter, editOrderButtonGetter ],
```

Comment:
removed `edit order` button and replaced with `payment status` because when redirected from a universal link, the edit button is not displayed on the order screen